### PR TITLE
Fix error shadow and error check warnings

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by hqhq** [comment]
LGTM

File: delete.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 )
 
 func killContainer(container libcontainer.Container) error {
-	container.Signal(syscall.SIGKILL, false)
+	_ = container.Signal(syscall.SIGKILL, false)
```
Comment (crosbymichael) [inline delete.go]: Its this golint or govet stuff?

**Comment by crosbymichael** [comment]
LGTM

File: delete.go (hunk 1 above)
Comment (hqhq) [inline delete.go]: I think neither, @mrunalp how did you get these warnings?

Comment (mrunalp) [inline delete.go]: I ran gometalinter. I hope that we could enable it for runc at some point.

**Pull request merged**